internal/test-utils/src/jvmMain/kotlin/coil3/test/utils/bitmaps.jvm.kt

```diff
@@ -0,0 +1,76 @@
+package coil3.test.utils
+
+import coil3.size.Size
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.async
+import kotlinx.coroutines.runBlocking
+import org.jetbrains.skia.Bitmap
+
+val Bitmap.size: Size
```

Comment:
Thanks for converting these! Could we create a common `Bitmap` type (only for our tests)? Looks like something like this should be enough:

```kotlin
expect class Bitmap {
    val width: Int
    val height: Int

    fun getPixels(): ByteArray
}
```

This way we can put the rest of these functions in `commonMain`, which will be useful for migrating more of Coil's tests to `commonMain`.